app/components/channel_list_row/index.tsx

```diff
@@ -117,10 +119,11 @@ export default function ChannelListRow({
                         style={style.icon}
                     />
                     <View style={style.titleContainer}>
-
                         <Text
                             style={style.displayName}
                             testID={channelDisplayNameTestID}
+                            ellipsizeMode='tail'
+                            numberOfLines={1}
```

Comment:
is this intended to add an ellipsis to the channel display name? again do share a screenshot with all affected views that use this component